htdocs/admin/mails_templates.php

```diff
@@ -119,23 +119,33 @@
 $tabname = array();
 $tabname[25] = MAIN_DB_PREFIX."c_email_templates";
 
+// Used by the GUI during creation so we don't ask the user to fill in tms and datec
```

Comment:
I don't think we need to introduce another table.
We know that tms and datec are date update and creation and should not be requested.
We can just add a hard coded if on this 2 fields when we forge the form to not show them.